diesel_cli/src/database.rs

```diff
@@ -303,19 +320,22 @@ mod tests {
 
     #[test]
     fn split_pg_connection_string_returns_postgres_url_and_database() {
-        let database = "database".to_owned();
-        let base_url = "postgresql://localhost:5432".to_owned();
-        let database_url = format!("{}/{}", base_url, database);
-        let postgres_url = format!("{}/{}", base_url, "postgres");
-        assert_eq!((database, postgres_url), change_database_of_url(&database_url, "postgres"));
-    }
-
-    #[test]
-    fn split_pg_connection_string_handles_user_and_password() {
-        let database = "database".to_owned();
-        let base_url = "postgresql://user:password@localhost:5432".to_owned();
-        let database_url = format!("{}/{}", base_url, database);
-        let postgres_url = format!("{}/{}", base_url, "postgres");
-        assert_eq!((database, postgres_url), change_database_of_url(&database_url, "postgres"));
+        // format: (original_string, dbname, expected_changed_string)
+        let test_cases: [(&'static str, &'static str, &'static str); 11] =
+            [ ("postgresql://", "", "postgresql:///postgres")
+            , ("postgresql://localhost", "" ,"postgresql://localhost/postgres")
+            , ("postgresql://localhost:5433", "", "postgresql://localhost:5433/postgres")
+            , ("postgresql://localhost/mydb", "mydb", "postgresql://localhost/postgres")
+            , ("postgresql://user@localhost", "","postgresql://user@localhost/postgres")
+            , ("postgresql://user:secret@localhost", "", "postgresql://user:secret@localhost/postgres")
+            , ("postgresql://other@localhost/otherdb?connect_timeout=10&application_name=myapp", "otherdb", "postgresql://other@localhost/postgres?connect_timeout=10&application_name=myapp" )
+            , ("postgresql:///mydb?host=localhost&port=5433", "mydb", "postgresql:///postgres?host=localhost&port=5433")
+            , ("postgresql://[2001:db8::1234]/database", "database", "postgresql://[2001:db8::1234]/postgres")
+            , ("postgresql:///dbname?host=/var/lib/postgresql", "dbname", "postgresql:///postgres?host=/var/lib/postgresql")
+            , ("postgresql://%2Fvar%2Flib%2Fpostgresql/dbname", "dbname", "postgresql://%2Fvar%2Flib%2Fpostgresql/postgres")
+            ];
```

Comment:
I think that's the only place this array formatting is used in this code base. Can you change it to trailing commas and the opening bracket on the same line as the `=`?